src/main/docker/Dockerfile

```diff
@@ -2,6 +2,11 @@ FROM eclipse-temurin:11.0.14.1_1-jre-focal@sha256:246f0a07f7ba2c52b48d1879aa6f29
 
 FROM debian:bullseye-20220125-slim@sha256:4c25ffa6ef572cf0d57da8c634769a08ae94529f7de5be5587ec8ce7b9b50f9c
 
+# Install wget for healthcheck
+RUN apt update \
+    && apt install -y wget \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Oh, good call! yeah I think that moving the `RUN` in-front of the `COPY` instruction certainly would help in local development as the other layers would be cached then.
From a CI perspective there is no gain but the development experience alone would make this definitely worth it!

I would move both `COPY` instructions though, while I am not 100% sure if the multi stage build would benefit from the layer cache (I assume so though), it would group the instructions.